Pachinko/Source.c

```diff
@@ -1,18 +1,46 @@
-#include <stdio.h>
 #include <ctype.h>
+#include <stdbool.h>
+#include <stdio.h>
 
+#define BEGINNING_CREDITS (1000)
+#define JACKPOT_MULTIPLIER (3)
 #define LINESIZE (1024)
 #define MINIMUM_CREDITS (100)
-#define BEGINNING_CREDITS (1000)
 #define NUMBER_OF_SYMBOLS (5)
 #define NUMBER_PER_LINE (3)
-#define JACKPOT_MULTIPLIER (3)
 
-int credits;
+size_t credits;
 int playedCredits;
-char symbols[NUMBER_OF_SYMBOLS];
 char spunSymbols[NUMBER_PER_LINE];
+char symbols[NUMBER_OF_SYMBOLS];
 
+/* 
+
+enum eSymbol
```

Comment:
You did it right.
Speaking of the use of this, I'd like to say try using this when rendering symbols using switch statement

Reply:
Ohh... Can I just keep my implementation? Seems cleaner imo, disregarding the declaration of an enum vs declaring all my symbols. You didn't answer how I would use my original symbols though. Like my ^ %  * don't really seem usable, or at least I can't figure out how to use them.

Reply:
1. You can keep, but I usually prefer enum over your method since when going over code, it's better to write like
switch(playerSymbols[0])
{
    case X:
        printf("x ");
        break;
}
So i can understand more quick.
(Note: enum is like macro. In here, X mean 1(i.e. X is replaced with 0))

2. I don't quite understand use of the symbols. Can you elaborate on this?

Reply:
Ahh okay I see how switch works yeah. And this also reinforces my understanding of enums...

However, I'm imagining this as a real slots machine, and slots usually have a bunch of different symbols right? So I want to be able to store and print out those symbols in my machine. My 5 symbols are ^ % * X and 7. Would I be able to store these in an enum and print these out to the user in the same fashion that I've done it now?

Reply:
Oh okay, I see that it's possible but I feel like it's a lot of code to write for something like this... 

I think I'll keep my implementation for now, and just keep enums in mind. I may change it later but I'm going to take a break from working on this.